README.md

```diff
@@ -42,7 +42,24 @@ In order to use the rules (or provided configurations), activate them via the lo
 
 The `custom` namespace used by the shipped [bpmnlint extension](./bpmnlint-plugin-custom) is arbitrary, i.e. can be changed freely. However you'd need to take into account how the linting infrastructure discovers rules and configuration:
 
-* Given a namespace `{FOO}`, it searches for a library `bpmnlint-plugin-{FOO}` in the NodeJS search path (usually `node_modules` folder)
+* Given a namespace `{FOO}`, it searches for a library `bpmnlint-plugin-{FOO}` in the NodeJS search path (usually `node_modules` folder. 
+
+    *Note - that when using plugin namespaces within `.bpmnlintrc` you exlude the `bpmnlint-plugin-` prefix from the name. Example usiting `using bpmnlint-plugin-foo` namespace*

```

Comment:
What about making this a dedicated section ("discovery of scoped packages". Could be less confusing.